validphys2/src/validphys/pseudodata.py

```diff
@@ -267,6 +268,9 @@ def make_replica(
         # positivity control
         if np.all(shifted_pseudodata[full_mask] >= 0):
             return shifted_pseudodata
+        elif "POL" in fitbasis and trials == (max_tries // 100):
```

Comment:
The "if POL" in fitbasis scattered around the code make me nervous.

Could you do instead something like `fitbasis.startwith("POL_")` or whatever so that it becomes a special key?

Otherwise at some point someone might do "POLYNOMIAL" or whatever and will be very confused ^^U

Reply:
Yep, I will do so xD

Reply:
Maybe there is even a more elegant way to pass this information around.

Reply:
Yes!

The "right" way to do it, or, more precisely, the way we have been doing it until now, is to define some processes which allow for negative values in the data.

For instance, at the moment is only the `ASY` types. You can (you should, actually!) add a new process type to the library of process options for the polarized dataset that will allow for this. The check is done a bit above in `if ASY in commondataproc`.

Reply:
For this, should I cal the new process `DIS_POL` such that everything such as `_get_ported_kinlabel` and `_get_process_description` still works out of the box? In this way, the changes will be minimal.

Reply:
This btw has been addressed here [0b0a925](https://github.com/NNPDF/nnpdf/pull/1816/commits/0b0a92522b0d79eb766d8dbd4764ce4e6c1f56c1) so I will remove that `elif` block.